packages/@aws-cdk/aws-codebuild/test/integ.defaults.lit.ts

```diff
@@ -0,0 +1,23 @@
+import cdk = require('@aws-cdk/cdk');
+import codebuild = require('../lib');
+
+class TestStack extends cdk.Stack {
+  constructor(parent: cdk.App, id: string) {
+    super(parent, id);
+
+    /// !show
+    new codebuild.Project(this, 'MyProject', {
+      buildSpec: {
+        version: '0.2',
+        phases: { build: { commands: [ 'echo "Hello, CodeBuild!"' ] } }
```

Comment:
Can you indent this nicely instead?

```ts
      buildSpec: {
        version: '0.2',
        phases: {
          build: {
            commands: [
              'echo "Hello, CodeBuild!"',
            ],
          },
        },
      },
```

I think it makes a better job of visualizing the buildspec structure clearly.

Reply:
SUre